script/script_hooks/destinations

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+if [ -z "$DEFAULT_OSX_DESTINATION" ]; then
```

Comment:
Could you move the env variable declarations to https://github.com/guidomb/ios-scripts/blob/master/install#L64